src/main/kotlin/com/sourcegraph/cody/ui/AutoGrowingTextArea.kt

```diff
@@ -28,11 +24,10 @@ class AutoGrowingTextArea(private val minRows: Int, maxRows: Int, outerPanel: JC
   val textArea: JBTextArea
   val scrollPane: JBScrollPane
   private val initialPreferredSize: Dimension
-  private val autoGrowUpToRow: Int
-  val undoManager = UndoManager()
+  private val autoGrowUpToRow: Int = maxRows + 1
```

Comment:
what changed here?

Reply:
Just an IntelliJ suggestion accepted to simplify the code (inline assuagement and variable which could be private).